epaint/src/text/font.rs

```diff
@@ -258,8 +254,15 @@ impl Font {
     }
 
     /// All supported characters
-    pub fn characters(&self) -> &BTreeSet<char> {
-        &self.characters
+    pub fn characters(&mut self) -> &BTreeSet<char> {
+        if self.characters.is_none() {
```

Comment:
Put `characters` behind a `epaint::RwLock` so that this function can be `&self` again.